aloha-io-avro/src/test/scala/com/eharmony/aloha/audit/impl/avro/AvroScoreAuditorTest.scala

```diff
@@ -465,4 +438,31 @@ object AvroScoreAuditorTest {
 
   val M1 = com.eharmony.aloha.id.ModelId(1, "one")
   val M2 = com.eharmony.aloha.id.ModelId(2, "two")
+
+  /**
+    * Serializing round-trip ensures that the data adheres to the schema.
+    * @param schema an Avro schema used to validate the data
+    * @param records records to serialize.
+    * @return
+    */
+  private[avro] def serializeRoundTrip(schema: Schema, records: GenericRecord*): Seq[GenericRecord] = {
```

Comment:
Moved this to the companion object and changed scoping so it can be reused in other tests inside `avro` package.